app/javascript/crayons/MentionAutocompleteTextArea/UserListItemContent.jsx

```diff
@@ -0,0 +1,33 @@
+import { h, Fragment } from 'preact';
+import PropTypes from 'prop-types';
+
+/**
+ * Component used to display user details in the MentionAutocomplete dropdown
+ *
+ * @param {object} props
+ * @param {object} props.user The user data to populate the list item content with
+ */
+export const UserListItemContent = ({ user }) => (
```

Comment:
I can't tell if there are changes here or was this component just extracted from the mention autocomplete component?

Reply:
no changes, just moved to a new file - CodeClimate has been pointing out that the `MentionAutocompleteTextArea` is really long - when I moved the `mergeRefs` code in there, I moved this out since it didn't seem like it had to stay in there